### DISCUSSION TITLE
Would it make sense that this would be implanted in javascript?

### DISCUSSION THREAD

**Discussion by billsmithsmithbill22** [nicksawhney/bernie-sits#31] [Ideas]
If it ran in javascript, it could just be hosted on GitHub pages for free.

**Comment by billsmithsmithbill22** [comment]
Also, have you looked at the openstreetcam api? It's completely open source and free. 
[(http://api.openstreetcam.org/api/doc.htm)]

**Comment by samarmohan** [comment]
No this would not make sense for many reasons.
1. The API (Google Maps) is the one costing room much, so changing anything would not help.
2. @nicksawhney needs a backend to do this stuff (at least he said it’s better, issue #4) and you cannot host a Node.js backend on GH Pages.
3. Any maybe openstreetcam api is good but are you sure it can withstand a prod environment? I think waiting for funding is better.

Note: this should have been an issue instead of a Discussion

**Comment by nicksawhney** [comment]
@samarmohan This repository is actually not for the maintenance of the _deployed_ version of the site, rather it's an open source project based on the original site. No issues necessary for cost-related things (unless it's #25 ) Implementations of different APIs can be submitted as pull requests, but the priority here is building a cool piece of tech rather than all of the financial focuses. It's up to individuals deploying the site to figure out their own creative ways of funding! Let's keep the discussion focused. Thanks!
